tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md

```diff
@@ -0,0 +1,527 @@
+---
+author_name: Iwona Hahn
+author_profile: https://github.com/iwonahahn
+title: Create an SAP Fiori Elements-Based Analytical UI for your CAP Application
+description: This tutorial shows you how to create an analytical page using SAP Fiori elements app on top of your previously created CAP application.
+keywords: cap
+auto_validation: true
+time: 25
+tags: [ tutorial>beginner, software-product-function>sap-cloud-application-programming-model, programming-tool>node-js, software-product>sap-business-technology-platform, software-product>sap-fiori-tools, software-product>sapui5]
+primary_tag: software-product-function>sap-cloud-application-programming-model
+---
+
+
+
+## Prerequisites
+ - [Prepare Your Development Environment for CAP](btp-app-prepare-dev-environment-cap)
+ - Before you start with this tutorial, you have two options:
+    - Follow the instructions in **Step 16: Start from an example branch** of [Prepare Your Development Environment for CAP](btp-app-prepare-dev-environment-cap) to checkout the [`cap-roles`](https://github.com/SAP-samples/cloud-cap-risk-management/tree/cap-roles) branch.
```

Comment:
We want people to complete our tutorials. So, I would not tell them right away that they can skip a whole group of tutorials, and simply add this as a side note to your second option. Or you could put it as a note below the prerequisites section.

Reply:
Hi @MichaelCzcz,

I left another comment below, too. I'll be opening another PR and will transfer your comment there.

Best regards,
Svetoslav